capture.md

```diff
@@ -0,0 +1,111 @@
+# _Capture_
+
+The **_first_ step** in creating
+a **system** of for personal effectiveness
+is to help people ***capture***
+_everything_ on their mind. <br />
+Emptying one's mind (_specifically working memory_)
+of everything that is occupying space and processing power
+is _essential_ for gaining focus
+on the _one_ most important priority.
+This step is often referred to as a "brain dump".
```

Comment:
I would add a note here also about how having a trusted place to keep all of your thoughts is the only way to truly empty your mind. If you have scattered post-it's or a system that makes it difficult to view what needs to be done, the fear of forgetting something keeps everything in RAM anyway.

Reply:
@iteles yes, please refine as you see fit. 📝 
My objective was just to get _something_ down for the PM/PO to work with. 👍